wordpress-mobile/WordPress-iOS
# Issues/reader comments

This PR refreshes the reader comment table view cells UI, updating the style to match what is currently in the Calypso reader and replacing the DTCoreText backed WPRichTextView with our new WPRichContentView.

Fixes: #4034
Fixes: #4710
Fixes: #4641
Fixes: #6083
Refs: #5594

To test:
Take the ReaderCommentViewController for a test drive and confirm that all the following work as expected:
- Comments still load :)
- Threaded comments are properly nested up to 4 levels deep.
- You can like a comment.  A liked comment correctly shows its status. 
- You can reply to a comment.
- The comment author can be tapped to view the author's blog when the author's URL is available. 
- Comments still load more as expected.
- The post header and the reply text view are resized when changing orientation, and when multitasking on the iPad. 
- Links in comments are opened correctly.
- Images in comments display correctly and can be previewed. 
- Videos in comments display correctly and can be played. 

Needs review: @kurzee would you be game for the code review?  
cc @jleandroperez 

Once this one lands, there will be a final PR to remove DT and the last of its dependencies. :)

### Before: 

![simulator screen shot nov 30 2016 4 56 49 pm](https://cloud.githubusercontent.com/assets/1435271/20774923/262bd3d6-b71e-11e6-96b1-bdc5d7f551fd.png)


### After: 

![simulator screen shot nov 30 2016 4 55 15 pm](https://cloud.githubusercontent.com/assets/1435271/20774920/2216dc32-b71e-11e6-82ed-c5f6d418f0dd.png)
